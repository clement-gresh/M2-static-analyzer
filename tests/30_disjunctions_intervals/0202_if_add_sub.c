{
  int x;
  int y;
  int z;
  x = rand(10,20);
  y = rand(0,10);
  if(y>5){ x = y - x; }
  else { x = y + x; }
  z = 100/x;
  print_all;
}