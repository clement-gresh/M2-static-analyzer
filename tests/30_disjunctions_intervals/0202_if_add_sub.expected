tests/30_disjunctions_intervals/0202_if_add_sub.c:10.2-12: { âŠ¥ }, { [ x in [10;25], y in [0;5], z in [4;10] ] }
analysis ended